Swap76/NodeJS_Mongo_BoilerPlate
# add, configure and fix linting issues

adds eslint and precommit hook.

To test this pr manullay, 
1. introduce a linting error in any `.js` file
2. try committing your change
3. notice the linting error
4. to fix the error, run `npm run lint`
